app/controllers/AdminController.php

```diff
@@ -0,0 +1,137 @@
+<?php
+//require_once 'models/PostModel.php';
+//require_once 'models/CommentModel.php';
+//require_once 'models/UserModel.php';
+namespace app\controllers;
+use app\models\CommentModel;
+use app\models\PostModel;
+use app\models\UserModel;
+use app\utils\View;
+
+class AdminController {
+
+    private PostModel $postModel;
+    private CommentModel $commentModel;
+    private UserModel $userModel;
+
+    public function __construct() {
```

Comment:
@tony-tang-aligent as a general note it'd be nice to see some commenting around the class & methods. The standard for PHP is https://phpdoc.org/ and PHPStorm can auto-generate comments for you once you type:

```
/**
```

and then hit <kbd>Enter</kbd> in the line above a class/method/etc